assets/javascripts/discourse/initializers/extend-for-translate-button.js.es6

```diff
@@ -26,6 +26,16 @@ function initializeTranslation(api) {
     return;
   }
 
+  let nonRestrictedGroups =
+    siteSettings.restrict_translation_by_group.split("|");
+  let authorized_group = currentUser.groups.any((group) =>
+    nonRestrictedGroups.includes(group.id.toString())
+  );
+
+  if (!authorized_group) {
```

Comment:
```suggestion
  let authorizedGroup = currentUser.groups.any((group) =>
    nonRestrictedGroups.includes(group.id.toString())
  );

  if (!authorizedGroup) {
```

Tiny code-style note, we use camel-cased variables in Javascript. (Sometimes you will see properties using underscores, but that is usually for objects received from the server.)

Reply:
Super true, bad habits of almost never writting javascript, will take this into account.